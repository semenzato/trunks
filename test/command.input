_TPM_COMMAND TPM_VerySimple
_COMMENT TPM_VerySimple is the simplest kind.  There can be nothing simpler.
_IN_PARAM TPM_TAG tag TPM_TAG_RQU_COMMAND
_IN_PARAM UINT32 paramSize Total number of input bytes including paramSize and tag
_IN_PARAM TPM_COMMAND_CODE ordinal Command ordinal: TPM_ORD_VerySimple
_OUT_PARAM TPM_TAG tag TPM_TAG_RSP_COMMAND
_OUT_PARAM UINT32 paramSize Total number of output bytes including paramSize and tag
_OUT_PARAM TPM_RESULT returnCode The return code of the operation.

