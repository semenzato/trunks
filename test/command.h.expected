/* Copyright (c) 2011 The Chromium OS Authors. All rights reserved.
 * Use of this source code is governed by a BSD-style license that can be
 * found in the LICENSE file.
 */

#ifndef TRUNKS_COMMAND_H_
#define TRUNKS_COMMAND_H_

#include "g_tpm_structures.h"
#include "trunks_internal.h"

/* Sends a TPM_VerySimple command to the TPM and reads the response. */
uint32_t Ttl_TPM_VerySimple(void);

#endif  /* TRUNKS_COMMAND_H_ */
